ee/docker-ee/ubuntu.md

```diff
@@ -156,9 +156,11 @@ and update Docker Engine - Enterprise from the repository.
 6.  Use the following command to set up the **stable** repository. Use the
     command as-is. It works because of the variable you set earlier.
```

Comment:
Change the wording to read: "Set up the **stable** repository, using the following command as-is (which works due to the variable set up earlier in the process)."